packages/DecorativeIcon/__tests__/SVGIcon.spec.jsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react'
+import { shallow, render } from 'enzyme'
+import SVGIcon from '../SVGIcon'
+import { BatteryCharging } from '../svgs'
+
+describe('SVGIcon', () => {
+  const defaultProps = {
+    variant: 'default',
+    size: 24,
+  }
+  const doShallow = (props = { symbol: 'artificialIntelligence' }) =>
+    shallow(<SVGIcon {...defaultProps} {...props} />)
+
+  it('renders', () => {
+    const icon = render(<SVGIcon variant="default" symbol="artificialIntelligence" />)
+
+    expect(icon).toMatchSnapshot()
+  })
+
+  it('is hidden from screen readers', () => {
```

Comment:
There should be unit tests for size and variant as well.

Reply:
Done